datastore.js

```diff
@@ -123,4 +119,69 @@ var find = exports.find = function(object, tableName) {
             return resolve(docs);
         })
     })
-}
\ No newline at end of file
+}
+
+var insert = exports.insert = function(object, tableName) {
+    return new Promise ((resolve, reject) => {
+        var db = {};
+            db.schema = new Datastore({ filename: `${__dirname}/datastore/local/${tableName}`, autoload: true });
+            db.schema.insert(object, (error, newDoc) => {
+                if (error) {
+                    return reject(error)
+                }
+                return resolve(newDoc);
+            })
+    })
+}
+
+var update = exports.update = function(object, query, updateModifier, options, tableName) {
+    return new Promise ((resolve,reject) => {
+        var db = {};
+            db.schema = new Datastore({ filename: `${__dirname}/datastore/local/${tableName}`, autoload: true });
+            db.schema.update(query, updateModifier, options, (error, settingReplaced, affectedDocuments, upsert) => {
+                if (error) {
+                    return reject(error)
+                }
+            return resolve(settingReplaced, affectedDocuments, upsert);
+            })
+    })
+}
+
+var remove = exports.remove = function (object, options, tableName) {
+    return new Promise ((resolve,reject) => {
+        var db = {};
+            db.schema = new Datastore({ filename: `${__dirname}/datastore/local/${tableName}`, autoload: true });
+            db.schema.remove(options, (error, numRemoved) => {
```

Comment:
@aviquez96 object variable is not used.. Probably meant to be passed to `db.schema.remove`??